community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,909 @@
9jUZgPIHhJavOKAXpx4/Xzv3JCuQPTLaH9gQIEOg+gby8+2KqVkS5HorS1Voy0RIgQIAAAQIECBAgQIAAAQIECBAgQIBAawVWD0bKl7V2TKPdKVDP7yFBgACBygvkeHDlc5jqBFK+cSwEO6WneiHMT4AAAQIECBAgQIAAAQIECBAgQOAAAcd3H4DhQwJtFhhasm1VpPSLbZ6m74bPkb+3eeO6z/Vd4hImQKCnBBYvedmcktCynkpqCpJJ2U7pKWA3JQECBAgQIECAAAECBAgQIECAAIEjCzi++8g+7hJolcDi015xckq1v2rVeMY5QCCnfzzgMx8SIECgkgK1mPG4VF6VDL6Lgh4Nz5TuouUQCgECBAgQIECAAAECBAgQIECAAAECBAh0TOChL55WGxz8SKSY3rE5+2iifSP1v++jdKVKgECPCuQUj+3R1DqaVq4pSncU3GQECBAgQIAAAQIECBAgQIAAAQIECBAg0B0Cw3tnv7FEsrQ7oumtKHLOn9t23bobeisr2RAg0I8CKdKT+zHvVuecb9/nmdKtRjUeAQIECBAgQIAAAQIECBAgQIAAgWYFPFO6WUH9CRxZYGjpyt8sz5E++8it3G1UIOX0D4321Y8AAQLdIjD/AStOLKdpPLBb4qlwHLu3fueyHWPx1yqchNAJECBAgAABAgQIECBAgAABAgQI9JyAZ0r33JJKqIsE5p+4Yn7Z+fa+Lgqpx0LJezZtz+/tsaSkQ4BAHwrMGKg9tQ/Tbn3KOV9716CK0ndJ+JsAAQIECBAgQIAAAQIECBAgQIAAAQIEelkgTZ9b+1DZ+baol5Ocytzy2C7pH6+9fSpjMDcBAgRaIZBr8bRWjNPvY5Tncm+8y0BR+i4JfxMgQIAAAQIECBAgQIAAAQIECBAgQIBAzwoMLV11UTke/zd6NsEuSGw06pd1QRhCIECAQFMC8087577l58VjmhpE558K5J8XpQeZECBAgAABAgQIECBAgAABAgQIECBAgACBXhYYXrby90p+5/ZyjlOdW87x2Vs3rLv7mNapjsf8BAgQaFRg2kB+UaN99bunQE7JTul7kviMAAECBAgQIECAAAECBAgQIECAAAECBHpRYOEDVy4pef1zL+bWTTmlnN/aTfGIhQABAo0JrB5MKf9xY331Olhg/8jPi9J2Sh+s43MCBAgQIECAAAECBAgQIECAAAECBAgQ6AmBhfc/a960WvpkSeaonkioW5PIeeumjT/+QLeGJy4CBAhMVGDxku1/lCKdMNH22h1RYPe2ay/5wV0tPFP6Lgl/EyBAgAABAgQIECBAgAABAgQIECBAgEDvCDz0xdMGZ07710jxC72TVJdmkuMdEVeOdml0wiJAgMAEBVbXUopXT7CxZuMJ5Hz30d1jTe2UHg/MfQIECBAgQIAAAQIECBAgQIAAAQIE/n/27gXMsqo+EP3a1XQD3Q1+4ww04Vb5zTgSUARGGob4eb+BzNxJ4N5Jro6PGIyJjo6T6IctRgEVBQyoPKt5qFGCRgO+MWp00lFvQicaQ7oLQtMtoNgiXUB3ddOP6np0vc66a+3T1bTQz6pTVeec/dvYdV57r73+v31e7v9Z60+AQKsJFMePLv5aSi78n63W8VbrbwxhvFaLt7Rav/WXAAECzxY47qX9ry2K4qRn3+/2VAWKtXtvKSm9t4brBAgQIECAAAECBAgQIECAAAECBAgQINDyAkteevHtKYjfbvlAWiGAGL+45aHlT7VCV/WRAAEC+xV40UVHpumlr9/v4x44bIFaEf5h740kpffWcJ0AAQIECBAgQIAAAQIECBAgQIAAAQIEWlpgyakXXx6K4i0tHUQLdT5OTPxJC3VXVwkQILBPgeOPnp+n7VbuYZ86U7uzI8S/33tLNaX31nCdAAECBAgQIECAAAECBAgQIECAwBwLFEruzfERsPtWFlhy6rveGjoKSdLZOogx/O/ND93y09nanf0QIEBgJgSOe/E7TypivGQm2q5qmzGGjZse7F6/d/yS0ntruE6AAAECBAgQIECAAAECBAgQIEBgjgVyfdY57oLdE2hJgeNOe/elsSg+3ZKdb9FOT8RwQ4t2XbcJECAwKVB0zDvirjTDxpGTd7icvkAR4j3PbsX03c8WcZsAAQIECBAgQIAAAQIECBAgQIAAAQIEWkmgOP6l7/54mmXgj1JSoZX63ep9Xbtl3U1/1+pB6D8BAtUWOP6lF78nFOHsais0PvoYil+aujvvQVK68c5aJECAAAECBAgQIECAAAECBAgQIECAAIHZEHjRRUcef/QRf5lS0RfMxu7s4xmBWoymSX+GwzUCBFpQ4LiXvPtl6fPjwy3Y9abv8njtl+tJ5w5LSjf9YdNBAgQIECBAgAABAgQIECBAgACBKgmkk6PO2VXpgIt1ygLH/Oof/5uFC2rfCaH4j1NuxIZTE4jxoc1ru78ytY1tRYAAgbkXWPyii44rOuJ30gwbR819b9quB9u2/rh73bOj8gX32SJuEyBAgAABAgQIECBAgAABAgQIEJhDgVxTWmJ6Dg+AXbeEwHGnvus/FB21b6eE9P/REh1us07WiuKKNgtJOAQIVEkgzbKx8Kg0y0ZR/EqVwp7FWL+/r3117OtO9xEgQIAAAQIECBAgQIAAAQIECBAgQIAAgWYUSPU//7Cj6Li3kJCem8MTw8ObH7zpq3Ozc3slQIDAdAWu7Dj+6PlfSQnpV0y3JdvvW2CiFr++r0eMlN6XivsIECBAgAABAgQIECBAgAABAgQIECBAoLkETr1ywfEdOz6bktEXNlfHKtabGC6tWMTCJUCgjQSWvHTHp9MsG7/dRiE1VygxjG7ZEb65r05JSu9LxX0ECBAgQIAAAQIECBAgQIAAAQIE5kjA1N1zBG+3TS1w3IvfeVJR7PhKSkj/h6buaJt3LoZ4X9+67m+1eZjCI0CgLQWu7Fjy0v7loQhvacvwmieo74be7uF9dUdSel8q7iNAgAABAgQIECBAgAA
```

Comment:
Please remove these additional resources (starting here, and continuing to the end of the file). ClusterServiceVersion does not support listing arbitrary cluster resources at this point.
Please provide a description of why these resources are required so that we can help to find a way of modifying the CSV to properly include these resources.